software/contrib/menu.md

```diff
@@ -0,0 +1,142 @@
+# menu
+
+author: Matthew Jaskula
+
+date: 2022/02/15
+
+labels: utility
+
+A Menu which allows the user to select and execute one of the scripts available to this EuroPi. The menu will includes
+the contents of the contrib directory as well as the calibrate script. In addition, handlers are added that allow the
+user to exit the running script by rebooting the board back to this menu. Both the selection of a menu item and the
+exiting of a running program is achieved with the a 'hold one button and tap the other' action. This will work with 
+either button first.
+
+In the menu: 
+
+* button 1: scroll up
+* button 2: scroll down
+* tap button 1 while holding down button 2: launch selected program
+* tap button 2 while holding down button 1: launch selected program
```

Comment:
This interface felt a little awkward to me. How do you feel about changing it to use a knob to scroll through the list of scripts, then a single button press to select? I imagine if the number of scripts gets large then repeatedly tapping the button to find a program could get tedious.